static/src/javascripts/lib/fetch.js

```diff
@@ -0,0 +1,86 @@
+// @flow
+import reqwest from 'reqwest';
+
+/**
+ * Provide a minimal function equivalent to fetch. I don't dare calling it a
+ * polyfill but the signature is the same, albeit simplified.
+ *
+ * fetch (input, init): Promise
+ *
+ * Differences with the standard fetch are
+ *
+ * - fetch
+ * - - input can only be a string, Request is not supported
+ * - - does not support JSONP
+ * - - headers can only be specified as an object literal, the Headers interface is not supported
+ * - Response
+ * - - blob is not supported
+ * - - formData is not supported
+ * - - headers are not populated
+ *
+ * If you're still wondering what it actually supports
+ * - CORS
+ * - credentials
+ * - body, any body that you want to add to your request except for GET or HEAD
+ * - response.text() and response.json()
+ * - response.ok .status .statusText
+ */
+const buildRequest = (path, options) => {
```

Comment:
Yeah sounds good. I guess we're currently only requiring it for exported functions?